java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -1903,9 +1903,33 @@ public ColumnVector asTimestamp(DType timestampType, String format) {
    * @return A new vector allocated on the GPU.
    */
   public ColumnVector asStrings() {
+    if (type.isTimestamp()) {
+      return asStrings("%Y-%m-%d %H:%M:%S");
+    }
     return castTo(DType.STRING);
   }
 
+  /**
+   * Method to parse and convert a timestamp column vector to string column vector. A unix
+   * timestamp is a long value representing how many units since 1970-01-01 00:00:00:000 in either
+   * positive or negative direction. This mirrors the functionality spark sql's from_unixtime.
+
+   * No checking is done for invalid formats or invalid timestamp units.
+   * Negative timestamp values are not currently supported and will yield undesired results. See
+   * github issue https://github.com/rapidsai/cudf/issues/3116 for details
+   *
+   * @param format - strftime format specifier string of the timestamp. Its used to parse and convert
+   *               the timestamp with. Supports %Y,%y,%m,%d,%H,%I,%p,%M,%S,%f,%z format specifiers.
```

Comment:
%z implies time zones and local are supported but I don't think that they are.
%f is not documented on the listed web page, but from the code, it is partial seconds.  It would be good to document this too.